src/guiapi/include/display_helper.h

```diff
@@ -9,12 +9,36 @@
 #define RENDER_FLG_WORDB 0x1000 // multiline text
 #define RENDER_FLG(a, r) (a | r << 8) // render flag macro (ALIGN and ROPFN)
 
+#define TEXT_ROLL_DELAY_MS 50
+#define TEXT_ROLL_INITIAL_DELAY_MS 4000
+
 #ifdef __cplusplus
 extern "C" {
 #endif //__cplusplus
 
+typedef enum {
+    ROLL_SETUP = 0,
+    ROLL_GO,
+    ROLL_STOP,
+    ROLL_RESTART,
+} TXTROLL_PHASE_t;
+
+typedef struct _txtroll_t {
+    uint8_t phase;
```

Comment:
reorder vars so that they can be better aligned - less unused space

Reply:
and be very careful where this struct gets included into other structs - beware of #pragma pack